docs/ref/forms/fields.txt

```diff
@@ -1215,6 +1215,9 @@ method::
         field will be derived, and which will be used to validate the
         user's selection.
 
+    ``ModelMultipleChoiceField`` also takes one optional argument,
```

Comment:
I agree with restructuring the docs. I removed the description for `queryset` and linked both arguments to their descriptions in `ModelChoiceField`.

I rebased and squashed the new commit with these changes.